src/main/java/teammates/common/datatransfer/questions/FeedbackTextQuestionDetails.java

```diff
@@ -57,7 +57,7 @@ public String getQuestionWithExistingResponseSubmissionFormHtml(boolean sessionI
             int responseIdx, String courseId, int totalNumRecipients, FeedbackResponseDetails existingResponseDetails) {
         return Templates.populateTemplate(
                 FormTemplates.TEXT_SUBMISSION_FORM,
-                Slots.DISABLED, sessionIsOpen ? "" : "disabled",
+                Slots.IS_SESSION_OPEN, sessionIsOpen ? "true" : "false",
```

Comment:
Since the `readonly` value needs to be set to `false` if the session is open and vice versa, I wonder if you agree that it would be better if you passed the data down as `IS_SESSION_CLOSED` instead. So your new line would read something like:
```Java
Slots.IS_SESSION_CLOSED, !sessionIsOpen ? "true" : "false"
```

Note: you would have to rename the Slots constant and its content accordingly.